server/src/main/java/com/cloud/vm/UserVmManagerImpl.java

```diff
@@ -1459,9 +1459,10 @@ public UserVm updateDefaultNicForVirtualMachine(UpdateDefaultNicForVMCmd cmd) th
     }
 
     @Override
-    public UserVm updateNicIpForVirtualMachine(UpdateVmNicIpCmd cmd) {
+    public UserVm updateNicIpMtuForVirtualMachine(UpdateVmNicIpMtuCmd cmd) {
```

Comment:
service level call renaming is not as bad as API but migyht break third party plugins as well!
Also I don't think it is needed, a small refactor and two methods, one calling the other will do the trick.